docs/admin/scale/gke.md

```diff
@@ -0,0 +1,62 @@
+# Scaling Coder on Google Kubernetes Engine (GKE)
+
+This is a reference architecture for Coder on [Google Kubernetes Engine](#). We regurily load test these environments with a standard [kubernetes example](https://github.com/coder/coder/tree/main/examples/templates/kubernetes) template.
+
+> Performance and ideal node sizing depends on many factors, including workspace image and the [workspace sizes](https://github.com/coder/coder/issues/3519) you wish to give developers. Use Coder's [scale testing utility](./index.md#scale-testing-utility) to test your own deployment.
+
+## 50 users
+
+### Cluster configuration
+
+- **Autoscaling profile**: `optimize-utilization`
+
+- **Node pools**
+  - Default
+    - **Operating system**: `Ubuntu with containerd`
+    - **Instance type**: `e2-highcpu-8`
+    - **Min nodes**: `1`
+    - **Max nodes**: `4`
+
+### Coder settings
+
+- **Replica count**: `1`
+- **Provisioner daemons**: `30`
+- **Template**: [kubernetes example](https://github.com/coder/coder/tree/main/examples/templates/kubernetes)
+- **Coder server limits**:
+  - CPU: `2 cores`
+  - RAM: `4 GB`
+- **Coder server requests**:
+  - CPU: `2 cores`
+  - RAM: `4 GB`
+
+## 100 users
+
+For deployments with 100+ users, we recommend running the Coder server in a separate node pool via taints, tolerations, and nodeselectors.
+
+### Cluster configuration
+
+- **Node pools**
+  - Coder server
+    - **Instance type**: `e2-highcpu-4`
+    - **Operating system**: `Ubuntu with containerd`
```

Comment:
Ubuntu version is missing. We could also refer to the VM image.

Reply:
Totally agree about including versions and dates! GKE refers this as `ubuntu_containerd`, which varies based on the Kubernetes version, so using this as a mental note to include the cluster version too ;) 

<img width="618" alt="Screen Shot 2022-12-01 at 3 26 28 PM" src="https://user-images.githubusercontent.com/22407953/205180490-18b66af7-001c-4358-9357-1c1c7e74649f.png">